### PR TITLE
Updates and fixes

### LINKED ISSUE #194: [Bug] Player List error when game menu is open
**Have you read our FAQ/Wiki/#before-you-ask ?**
Yes

**Describe the bug**
JS error logged to console regarding the Player List app. (Just something I noticed when working on a mod.)
```
CEF.MainGEUI#local://local/ui/modules/apps/BeamMP-PlayerList/app.js:130
Uncaught TypeError: Cannot read property 'rows' of null
```

Cause (?):
https://github.com/BeamMP/BeamMP/blob/100fd5bec259db8047fe9bb88f55b2881dd2076d/ui/modules/apps/BeamMP-PlayerList/app.js#L80
`playersList` is `null` because the node does not exist when the menu is open.

Solution (?):
Add a null check and return if `playersList` is `null`.

**To Reproduce**
1. Join a server
2. Use a UI with the Player List app added
3. Open the game menu (Esc)
4. JS error is logged to the console every second while the menu is open

**Expected behavior**
No error occurs

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS + Version: Windows 10 21H1
 - BeamMP 4.2.3
 - BeamNG 0.24.1.1

### PR DESCRIPTION
**Pull request by snepsnepsnep**

### CODE REVIEW

**Review by 20dka — APPROVED** [review]

**Review by Olrosse — APPROVED** [review]

File: lua/ge/extensions/MPVehicleGE.lua [hunk 1]
```diff
@@ -932,6 +932,7 @@ local function onServerVehicleCoupled(serverVehicleID, data)
 end
 
 local function onServerCameraSwitched(playerID, serverVehicleID)
+	if not players[playerID] then return end -- TODO: better fix?
```
Comment (Starystars67) [inline lua/ge/extensions/MPVehicleGE.lua:935]: No, this should do the job :)

File: lua/ge/extensions/MPCoreNetwork.lua [hunk 2]
```diff
@@ -334,7 +335,7 @@ local HandleNetwork = {
 	['V'] = function(params) MPVehicleGE.handle(params) end, -- Vehicle spawn/edit/reset/remove/coupler related event
 	['L'] = function(params) setMods(params) end,
 	['K'] = function(params) log('E','HandleNetwork','K packet - UNUSED') end, -- Player Kicked Event
-	['Z'] = function(params) launcherVersion = params; be:executeJS('setClientVersion('..params..')') end -- Tell the UI what the launcher version is
+	['Z'] = function(params) launcherVersion = params; end
```
Comment (Starystars67) [inline lua/ge/extensions/MPCoreNetwork.lua:338]: My only issue here is how does the UI know what the launcher/client version is?
We display this info in the bottom right on the main menu when you click on the game version.

**Review by Starystars67 — CHANGES_REQUESTED** [review]
Please see the comments I added in my review.

File: lua/ge/extensions/MPCoreNetwork.lua (hunk 2 above)
Comment (snepsnepsnep) [reply lua/ge/extensions/MPCoreNetwork.lua:338]: setClientVersion does not exist anywhere in the JS anymore so it was just throwing an error.
Currently the UI gets the launcher version through the getLauncherVersion() function in MPCoreNetwork.

**Review by Starystars67 — APPROVED** [review]

**Pull request merged**